src/components/sequencing/form/ArgTitle.svelte

```diff
@@ -96,12 +98,14 @@
       </div>
     {/if}
 
-    <div>Value Type</div>
+    {#if isSymbolAllowed}
```

Comment:
Is this allowed for all languages?

Reply:
Yes, currently.

Symbols are used to reference primitives in all of the currently languages we've seen so far. F-Prime has a design for complex arguments, but we don't support it yet.